deeplearning4j-keras/src/main/java/org/deeplearning4j/keras/DeepLearning4jEntryPoint.java

```diff
@@ -0,0 +1,45 @@
+package org.deeplearning4j.keras;
+
+import lombok.extern.slf4j.Slf4j;
+import org.deeplearning4j.nn.multilayer.MultiLayerNetwork;
+import org.nd4j.linalg.api.ndarray.INDArray;
+
+/**
+ * API exposed to the Python side. This class contains methods which are used by the python wrapper.
+ * It is instantiated directly in the server code.
+ */
+@Slf4j
+public class DeepLearning4jEntryPoint {
+
+    private final NeuralNetworkReader neuralNetworkReader = new NeuralNetworkReader();
+    private final NDArrayHDF5Reader ndArrayHDF5Reader = new NDArrayHDF5Reader();
+    private final NeuralNetworkBatchLearner neuralNetworkBatchLearner = new NeuralNetworkBatchLearner();
+
+    /**
+     * Performs fitting of the model which is referenced in the parameters according to learning parameters specified.
+     *
+     * @param entryPointFitParameters Definition of the model and learning process
+     */
+    public void fit(EntryPointFitParameters entryPointFitParameters) throws Exception {
+
+        try {
+            MultiLayerNetwork multiLayerNetwork = neuralNetworkReader.readNeuralNetwork(entryPointFitParameters);
+
+            INDArray features = ndArrayHDF5Reader.readFromPath(entryPointFitParameters.getTrainFeaturesFile());
```

Comment:
Is this how keras data is stored? Bit concerned about data representation among other things here..What about reading in numpy arrays?